axelrod/tests/strategies/test_human.py

```diff
@@ -130,3 +130,6 @@ def test_repr(self):
 
     def equality_of_players_test(self, p1, p2, seed, opponent):
         return True
+
+    def test_reproducibility_of_play(self):
```

Comment:
Is this a method in the parent class that needs to be overwritten? (Have looked there but don't seem able to find it.)

Reply:
Looks like maybe I renamed it to `test_clone_reproducible_play` (which has an exception for `Human`) so we can delete this